extra/1-currency-conversion.js

```diff
@@ -15,8 +16,10 @@ function convertToUSD() {}
   They have also decided that they should add a 1% fee to all foreign transactions, which means you only convert 99% of the £ to BRL.
 */
 
-function convertToBRL() {}
-
+function convertToBRL(brazilianGbp) {
```

Comment:
Your parameter naming makes this functions purpose difficult to understand. Try to use meaningful/semantic naming throughout your code so when others look at your code they have an idea of what it's supposed to do.  I would suggest using  "pound" or "amount".

Reply:
Thanks